### ISSUE TITLE
Design a coala README badge

### ISSUE THREAD

**Issue by palash25** [coala/meta#122]
Currently coala is mostly used by its own developers. I was thinking that it would be nice to have a badge which says something like *Linting by coala* or *Linted using coala*  that all the coala devs can add to the README.md of their projects that use coala for linting.

A few coalians (like adtac) have some popular projects that might help us get a wider user base.

**Comment by Man-Jain** [comment]
@palash25 Yes, We can write something like "Cutely Linted By coala" or a simple one like "Linted by coala bears" or "Linted with love by coala" 
This is a basic design we can add coala logo here https://img.shields.io/badge/style-Cutely_by_coala-green.svg?longCache=true&style=for-the-badge&label=Linted&logo=slack

**Comment by palash25** [comment]
> "Linted with :heart: by coala"

I like this one

**Comment by Man-Jain** [comment]
How's this https://img.shields.io/badge/style-With_♡%20by_coala-green.svg?longCache=true&style=for-the-badge&label=Linted&link=http://coala.io

**Comment by palash25** [comment]
I like it but can you make the :heart: red ?

Also I was thinking if we could have a small coala logo on the left side and then the text. But I guess thats too much work. Its okay for me if its simple as it is.

**Comment by Man-Jain** [comment]
Yes we surely can add a logo. But for now I don't know how to do it😅. They don't seem to have proper instructions regading it. But without it here its is https://img.shields.io/badge/style-With_❤%20by_coala-green.svg?longCache=true&style=for-the-badge&label=Linted&link=http://coala.io

**Comment by Man-Jain** [comment]
Looks like I didn't knew how to do it. It was pretty straight forward. Asked it in their channel and it worked.

https://img.shields.io/badge/style-With_❤%20by_coala-green.svg?longCache=true&style=for-the-badge&label=Linted&link=http://coala.io&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADwAAAA8CAYAAAA6/NlyAAALJElEQVR42u2aaVNTWRrH+QC+6G8w/Wqqxhdd/bILptu2qmecUXEBtVEUAUGBsEPYl7DJkkACwUBADRADUXaRNYCALCIyIKtsAgHEfW21bcf7zHkOXCpKEkMWxpniVP2LEG5yzu8+6zkXq62xNbbG1tgaq4OTbfcnT4ndMa9zB5O9JHbXODn2QxyJ/UtN4Xv4N7wGr8XP/M9BeuXYh2nChV50fJ1S5vEmr4H7O6q0K+GP6t7kDyj2vYwq37dRcuff2JtA5PlVw3tK9n3HkdhJWEhcvKwl4vf6fv6HnslcxlA13xH9W9Ee8x5vEH4PfudXAX5dJvumqThvR7lcuCdCelLOgqaUe77BRVMAE9UxJv6I1mfBnYUHvtl0SJUiP7ZJkTdABJqqUYiZmhbBR+MB9VsdXZ64+bxH9sG9mwLbWCw90HQp7znC6VNLtRR6xqSMJcAxPALyjrzCBGdRWNUlqT/CGCylFGoaBXC5OpEpqz3LVNSnMNVNaUzjDZFZ3HwludkpLOLiBMBZE6YyXwTnk2NAFBUMZwO9IMH3DGRGB4EiM3kduIBcE+lzGpJCfCGTFwb5qTwoFqdC9YUsvCnUG6435jEdXbnQNUJct0fEKArjoTAnmik4F8NUVaSCLvCMap+3BPqGWaHrS3K+1XTjovREEEUGgSDUH1KCOJDo7wE8b3eI9nCBMLcTkBTgCddk5z6BlqUngJzcjPI8ITRekur1jMvZaSAI84N4chOTyfenh/mDIi+e+QK0wmzAZIEF7GLkGUnEklwQhgdAKtdn1bqnIcbrFESePgkhro4Q5OQAPGJRvN4U4VyJfh50noyIAGhs1x0Kq7U7xSzWZRdQQawjjgkFYVQg8EP9IDnYGxL8zkCstxtEnnGBi8IUpretgVmeH2f6uxqZK1IRYyp0hVRIPSaNhINSnqQ3pjGRYfY2W6LKjY+ArOhg4mKr1iULifNxhxhPVxDFhjHqmVHm+eM5ePpwBh7fvwuTI90mA6MKBQnUk85nRH4xe2PJMime2VpblpMB4tgQEEaidX1pbCUQd4vlnCLWdYbQU8ch6ORR6GmtZZ4sT8LjpXF4oB6COoXULNCZJGdIkkKAhdMXz14S+xxTgAFVIIgnGTkI0sP9qXWT/FesG+3pAuHuJyA7MZqplOczZQU5DFr34cIILM8PQl2xeYBLJQIoOBdtUHOCHZlRbahKnv8zO2E2L5QkjkCMJepeBSJSTyuUzPVr5cw8ceUXT9Tw7NE9ePJgEh4tjcGDhSGYvdtDYc2lqjKBQTU6uzbwHVrZuK6KdakojF0/LEPUuqqqUua3V0/g1fNlePl0kcDOwtMH0wR2fNW6A9DTUmFWYKzXhjYlRlm5USHl0SYjTwTpq2UoKWClDDVVKpnXLx8S4Pvw/Mk8te7j5YkV66rvwP25f4GqosCswKiuQcPaVdy8eJ2z8zIK+FpBNttk0DLE47hBYZaAQeu+eLoAzx7PEleeIpl5nLjyMLXu4r0+UJXLzA7c2mgYMG5JcT+9IeDqS1lCdqLUYO/VMnSaJCpX/Mk8vD9DYneelqEnxLoPF0dXrDvbD4szvdBUdtHcwChsPy2TvGT5sUp2EuyZsV/WLENXS2TMSqKaoq78kFgXXXmJWFc91Q21lyQWAW5rNszK2H2RHZWjwcAXc3jj7CT5Z6NpGYrycCVlyAm4zscg4MQR6O9S0TL0iFgXyxBad2H6Jqgnu6C2yALAKKVhW0+syeR8LNxg4IKs+Nd1hRI6ydWLYogl/XIE6ZdDXY9D4EkHKM7LZDqbKpmVRDWE1iWufAsWiHWHe+sorKXU1vZlYGljCIlju1qDgeWipI/lORlrk4hjuXQ3FEysG+ruxNyfG/mkyViavU2Ab8L8ZCfUK6UWBVaVSsEQYNw6GgysEJ0FVH1hDp2ktkBCd0UBTr+C77FDpLdNYlYS1dBqorpFXflmaymFtbQ6unP1QleR01BMXBsGviIRrE1SlpMOQc5HwdvBjko9fZvBMoTWXZjuganhVqZOnmM0cEWeCA8MTGlETAdGVeWLNE87MFtT4KbKIuLaK2VoauQ6Y2qi+u4vf4aEhAQmIjyMKSLz6rqujnhbIenvW/vEOoEVHbHvjQZGVZ/PWtfQX+DHMd2qUqavrZKpKzE9bhFYIpEwy8vLzAk7W63XYGgJwlZ6erk0Vm8MbwhYnBrCfA6N28SGolyLxaXH8SOwbds2Zvv27Qy+1nYNnrhgxcC+ICPCH/QBu6buHjYYWJTGfYOQn6skKxWuXhBbDJpP9r5oXYxn9j3NHIJNT9QZZzxHo+dpDSoh6OqnT4v2tRoMzOcHzCGgLuEJZXeDEjquFuFiNkV4eIhlEZsfbHHjSat7KS9Oq4XxWZZT4q58g4F5Z93bEEyfhnpqYKCzatOAsa0NPPErhLg44qEhde2suKB1Fq7tF9Be2inxnxEGA5+JPBhnCHB/e8WmAWNl8HU8hCej6Np4eEgPEXUlLPvgHe4GA+/2sv5ewo/8+AVgFJ41bxowys/xMHBdjq65traNg1u67RQyWG1kBPOOLxgC3FomszgslkEW2OeoPXVtrqsj2aO7fwKsWt0aHg7bWWu10eEY/He5HmCMX9atLW7ljPAAFph1bbqJSY/x/QQ4Xun2BoH3B+yI2zAwukRMgvsrXcA9Tco1K3fVFZsEhGWoXq67xnNdHBH0E/kfPwwV5cnweTvpGLerlbqzMcMh6JcbuoAblFIKy6q7vsQk6Mr8THwupSt+10kQy9FuXd8fy6yMHbZe1s7R8aeeawO+IuHj3pdq5FYDVa+qDFTF+cZCY1ODD93Wvc/276ykEi5o653Runu9f/jZypRxwOfHZpkwltEG3VZdBGO3VUTNMN7fQtQKo+R1V50SmksuGAWNBw8l2WlQmpuON4BavYg0GDy/U5Ar5oKqV6i17npm71+09bZZl6yMimWX0D1TWltNcSqxrAruDrQRtcPEYAfRDZi80wmTQ10kodVCZ+1laC3feEemIG2s7Fwc1Kj4Onvm9rFs+iBtJVH91ENj1xxjr7e1SJdrV57PJJZtg6mhbqrpkZtUM6O3qO6N3SbqJ17QvrZwPGPuvJ0LmsJNPR7foPB39oRSHyz7b04OUX9rIWvk6QQwDtpmMIPPfacNuvGyjED2EsA+qnvj/VSzdweo5ibuEA1B9+AFXKzJwnrLwrqk7L5p623dZmXuccD/+2/2cWyWRGnBf2iDrinKJdC3CNggBZyfHCIaBvXUCNEoLEyPwzDxBFNhsfywbuzG3zuw18dmGNdmdmA2nhGan+KnFbosVwij/V2gnh4lGoOFmXGiu7B4b4JoEpZmpxHaaFh8qoCgqBNJuzpsOTZqzbi1GLQtx/qFrphGddZXEuuOEsApcoQ7Q7Wsnl3VHPT1VW3I0lh2cMvHwjpE/9K8l2M9R2E3Y/zD44dvMaaduLsXdZUsZTYfuhtrKCCr2YlR6L5z0WDQK+R/MtlYZUsPZmOc2wg3Nj2miUsVkNr3MirB/Zkua5dJRTDU27kGPdjfpDfzojXxxIKNUxTbVJDk9AIrhtV/c2Bng+51yG/nkj7wpitFJJ4nQT0zAfjgGvetKGwJUZouux7UZhTnoF3U1zL2cGz8cVH7ff76wDv6iDqbH/ZBm5tPDg+A5DLvHcLokrtw/8Qh7s6Wfd7WC/id2OJafa0DF4d1kVgFEN411HY2JM7pET/V7604Pex9tUwCxRWZ7zXhXNJ292Aisgv6qR0h8bMkMVaZGdTyiY3CkzjHJIMQ+oTX4LX4Gfys1f/DQBCMQ01RuK2xNbaGseM/9y+u2ItTYKkAAAAASUVORK5CYII

**Comment by palash25** [comment]
Looks good @Man-Jain just trying using a shortened url next time :wink:  Anyway thanks for this :smile: 

Lets wait for a maintainer and other devs to take a look and approve this.  cc @jayvdb

**Comment by li-boxuan** [comment]
Looks good! Btw coala is always with a small 'c' 😉

**Comment by Man-Jain** [comment]
@li-boxuan Yes I knew that but it was all Capital so had no other choice. 😅

**Comment by Man-Jain** [comment]
@palash25 Any updates regarding this ?

**Comment by palash25** [comment]
Only the maintainers can tell. @Makman2 likes the badge however so I guess this will be accepted in some time

**Comment by Man-Jain** [comment]
Okayy Nice 😊
